lumicks/pylake/force_calibration/detail/hydrodynamics.py

```diff
@@ -29,7 +29,7 @@ def calculate_dissipation_frequency(gamma0, bead_radius, rho_bead):
     Parameters
     ----------
     gamma0 : float
-        drag coefficient [mPas]
+        drag coefficient [m Pa s]
```

Comment:
I initially read this as milli Pas. I think it's better to separate the units to avoid confusion

Reply:
Yeah, makes sense. I have no objections to changing it.

It would be nice to restructure/clean up the commit history of this PR a bit without the review fix-up commits (squashing those changes into the respective commits where those things were added) and such. That way different changes can quickly be read off of the history.

Considering it's a pretty complex PR and some things changed a fair bit, it might be easier to quickly open a new branch and cherry-pick from this one (maybe after the +df fixup).

Reply:
All right, compressed into 4 commits. Shall I open a fresh PR as well? Or put this one to "ready for review"?